src/usr/share/batwatch/batwatch.py

```diff
@@ -53,124 +46,150 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
     def __ne__(self, other):
```

Comment:
I'm a little surprised Python requires you to define all these equality checks. I know C++ only requires you define less-than. It can derive the others from that.

Reply:
It does not. "There are no implied relationships between comparison operators."

https://docs.python.org/2.7/reference/datamodel.html#object.__ge__